app/src/pages/inside/profilePage/configExamplesBlock/configExamplesBlock.jsx

```diff
@@ -27,29 +27,29 @@ import { TabsConfig } from './tabsConfig';
 
 const cx = classNames.bind(styles);
 @connect((state) => ({
-  activeProject: activeProjectSelector(state),
+  projectName: projectNameSelector(state),
```

Comment:
Please provide `projectKey` instead as it will be used for reporting instead of `projectName`.

Reply:
Done.